pkg/agent/openflow/framework.go

```diff
@@ -171,20 +171,26 @@ func (f *featurePodConnectivity) getRequiredTables() []*Table {
 		L2ForwardingOutTable,
 	}
 
+	hasIP := false
 	for _, ipProtocol := range f.ipProtocols {
 		switch ipProtocol {
 		case binding.ProtocolIPv6:
 			tables = append(tables, IPv6Table)
 		case binding.ProtocolIP:
+			hasIP = true
 			tables = append(tables,
 				ARPSpoofGuardTable,
 				ARPResponderTable)
 			if f.enableMulticast {
 				tables = append(tables, PipelineIPClassifierTable)
 			}
-			if f.connectUplinkToBridge {
-				tables = append(tables, VLANTable)
-			}
+		}
+	}
+	if f.connectUplinkToBridge {
+		tables = append(tables, VLANTable)
+		if !hasIP {
+			tables = append(tables,
+				ARPSpoofGuardTable)
```

Comment:
Did not get why we need to do this. Could you explain? Maybe add some comments to code too.

Reply:
Added comments.
This is to allow Node's IPv4 traffic in an IPv6 only cluster and Nodes also have IPv4 network.